typescript/cli/src/utils/balances.ts

```diff
@@ -18,10 +19,31 @@ export async function assertNativeBalances(
         .getProvider(chain)
         .getBalance(address);
       const balance = ethers.utils.formatEther(balanceWei);
-      if (balanceWei.lt(minBalanceWei))
-        throw new Error(
-          `${address} has insufficient balance on ${chain}. At least ${minBalance} required but found ${balance.toString()} ETH`,
-        );
+      if (balanceWei.lt(minBalanceWei)) {
+        const symbol =
+          multiProvider.getChainMetadata(chain).nativeToken?.symbol ?? 'ETH';
+        const error = `${address} has insufficient balance on ${chain}. At least ${minBalance} required but found ${balance.toString()} ${symbol}`;
+        const isResume = await confirm({
+          message: `WARNING: ${error} Continue?`,
+        });
```

Comment:
My first thought was that like other confirm requests, this should skip if the user has included `--yes` flag. However, if they're doing that they should probably be made aware of balance issues so I guess let's leave this as is.